index.html

```diff
@@ -73,7 +73,7 @@
         <h3>Security</h3>
 
         <details>
-          <summary>Sandstorm protects you from evil (or buggy) apps while making sure security never gets in the way.</summary>
+          <summary>Sandstorm runs each app in a tight sandbox, keeping you safe from evil (or buggy) apps while making sure security never gets in the way.</summary>
```

Comment:
I got feedback from @geofft that the "sandbox" word may be meaningless to people. While that's true, I like saying something here that alludes meaningfully to the fact that our security isn't total nonsense.